adal/src/androidTest/java/com/microsoft/aad/adal/HttpDialogTests.java

```diff
@@ -70,21 +67,4 @@ protected void setUp() throws Exception {
                 .setBrokerPackageName(AuthenticationConstants.Broker.COMPANY_PORTAL_APP_PACKAGE_NAME);
         Log.d(TAG, "mTestSignature is set");
     }
-
-    public void testCreateDialogTest() throws NoSuchMethodException, ClassNotFoundException, IllegalArgumentException,
```

Comment:
you asked me to remove it, remember?

---
In reply to: [92568893](https://github.com/AzureAD/azure-activedirectory-library-for-android/pull/736#discussion_r92568893) [](ancestors = 92568893)